salt/reviewer-suggestions/init.sls

```diff
@@ -73,15 +85,39 @@ reviewer-suggestions-configure:
             - reviewer-suggestions-build-essential
             - python-dev
             - reviewer-suggestions-repository
+            - reviewer-suggestions-app-cfg
 
-reviewer-suggestions-preprocess:
+reviewer-suggestions-app-cfg:
+    file.managed:
+        - user: {{ pillar.elife.deploy_user.username }}
+        - name: /srv/reviewer-suggestions/app.cfg
+        - source: 
+            - salt://reviewer-suggestions/config/srv-reviewer-suggestions-{{ salt['elife.cfg']('project.branch') }}.cfg
+            - salt://reviewer-suggestions/config/srv-reviewer-suggestions-app.cfg
+        - template: jinja
+        - replace: True
+        - require:
+            - reviewer-suggestions-repository
+
+reviewer-suggestions-migrate-schema:
     cmd.run:
         - user: {{ pillar.elife.deploy_user.username }}
         - cwd: {{ pillar.reviewer_suggestions.installation_path }}/preprocessing
         - name: |
-            {{ pillar.reviewer_suggestions.installation_path }}/venv/bin/python ./updateDataAndReload.py
+            {{ pillar.reviewer_suggestions.installation_path }}/venv/bin/python ./migrateSchema.py
         - require:
             - reviewer-suggestions-configure
+            - reviewer-suggestions-server-service-stopped
+
+reviewer-suggestions-migrate-schema-and-start:
```

Comment:
usually for these fan-in states we use the past participle: `reviewer-suggestions-schema-migrated-and-started`